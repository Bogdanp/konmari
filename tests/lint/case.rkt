#lang racket/base

(define a 1)

(case 42
  [(1 2 3)
   (define a 1)
   (define b 2)
   (+ a b)]

  [(42)
   (define b 1)
   b])

(case 'b
  ['b 1])
