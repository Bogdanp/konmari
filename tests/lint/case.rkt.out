/case.rkt:7:12:warning:identifier 'a' shadows an earlier binding
/case.rkt:3:9:warning:identifier 'a' is never used
